scripts/list-data-directory.js

```diff
@@ -0,0 +1,29 @@
+/* global api, hashing, keyring, types, util */
+
+// run this script with:
+// yarn script listDataDirectory
+//
+// or copy and paste the code into the pioneer javascript toolbox at:
+// https://testnet.joystream.org/#/js
+// requires nicaea release+
+
+const script = async ({ api, joy }) => {
+  const ids = await api.query.dataDirectory.knownContentIds()
+
+  await Promise.all(ids.map(async (id) => {
+    let obj = await api.query.dataDirectory.dataObjectByContentId(id)
+    if (obj.isNone) { return }
+    obj = obj.unwrap()
+    console.log(`contentId: ${new joy.media.ContentId(id).encode()}, ipfs: ${obj.ipfs_content_id}`)
+  }))
+
+  console.error(`Data Directory contains ${ids.length} objects`)
```

Comment:
Shouldn't this be a `console.log`?

Reply:
I think here I was trying to isolate the json output from line 17 so we could redirect stdout to a file and leave out this log line from it.